apps/web/src/app/(private)/settings/ProviderApiKeys/index.tsx

```diff
@@ -65,7 +55,7 @@ export default function ProviderApiKeys({
                   <Button
                     size='small'
                     variant='destructive'
-                    onClick={() => destroyApiKey(apiKey.id)}
+                    onClick={() => destroy(apiKey.id)}
```

Comment:
I will give a try to simple action in forms for these simple UIs. You avoid having to use local stores